packages/web3/src/api/api-alephium.ts

```diff
@@ -340,18 +368,25 @@ export interface BuildMultisig {
   /** @format address */
   fromAddress: string
   fromPublicKeys: string[]
+  fromPublicKeyTypes?: string[]
+  fromPublicKeyIndexes?: number[]
   destinations: Destination[]
   /** @format gas */
   gas?: number
   /** @format uint256 */
   gasPrice?: string
+  /** @format group-index */
+  group?: number
+  multiSigType?: MultiSigType
```

Comment:
1. What's the default value for this?
2. if `fromPublicKeyTypes`, do we treat the address as the lagecy multisig always?

Reply:
1. The default value is `P2MPKH`
2. Since we want to reuse the existing multisig API, we introduced the `fromPublicKeyTypes` and `multiSigType` parameters. To use `P2HMPK`, users need to specify both `multiSigType` as `P2HMPK` and `fromPublicKeyTypes`

Correct me if I'm wrong @h0ngcha0